docs/howto/how-to-make-tensorflow-plugin-with-ui/tensorflow-plugin-with-ui.md

```diff
@@ -0,0 +1,1488 @@
+# 1. Create a Basic Text Classifier Neural Network with TensorFlow
+
+In this guide, we use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/), [Alteryx UI-SDK](https://github.com/alteryx/dev-harness), and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to illustrate how to create, train, and call a Tensorflow Keras Neural Network model in a workflow.
+To achieve this, We'll reference Tensorflow's [official tutorial](https://www.tensorflow.org/tutorials/keras/text_classification) and recreate the Basic Text Classifier as a workflow tool!
+
+Upon completing this guide, you will know how to:
+- Develop using the Python and UI SDKs
+- Exchange data between UI and Python SDK processes; during both `UPDATE` and `RUN` modes
+- Use metaprogramming, Python SDK, and the UI SDK to create an editable configuration and dynamic behavior between or during workflow runs
+- Package and use 3rd party libraries from `npm` and `pypi` in a tool
+- Develop "ready to use" Keras models from scratch, including models for deploying into production workflows.
+
+- [1. Create a Basic Text Classifier Neural Network with Tensorflow](#1-create-a-basic-text-classifier-nueral-network-with-tensorflow)
```

Comment:
We are missing a few sections.

You have more sections in the Guide then what is added in the Table of Contents